corehq/apps/app_manager/views.py

```diff
@@ -1266,13 +1270,14 @@ def default_new_app(request, domain):
         domain, _("Untitled Application"), lang=lang,
         application_version=APP_V2
     )
-    app.add_module(Module.new_module(_("Untitled Module"), lang))
-    app.new_form(0, "Untitled Form", lang)
+    module = Module.new_module(_("Untitled Module"), lang)
```

Comment:
it's a (very prevalent) convention but not built into the language or anything: http://commcare-hq.readthedocs.org/en/latest/translations.html